Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
Not sure.  What I do know is that this was causing several hundreds of etag update violations when running the 2nd script of the demo (network loader). Without reentrant, all the scripts run without issues. 
@jthelin is there a better fix?